### PR TITLE
SIP10 addition: add spec for memo

### PR DESCRIPTION
**Pull request by lgalabru**
This PR is a proposal for augmenting the transfer function with an optional memo field.
I'd also recommend going with a naming more explicit for this trait - something like `sip-010-trait` would be more descriptive: this trait is the first, but we will probably see other kind of fungible token in the future.

### CODE REVIEW

**Review by jcnelson — APPROVED** [review]
This LGTM! Just need @hstove's sign-off as the author.

File: sips/sip-010/sip-010-fungible-token-standard.md [hunk 1]
```diff
@@ -66,6 +66,18 @@ When returning an error in this function, the error codes should follow the same
 | u3         | `amount` is non-positive                        |
 | u4         | `sender` is not the same as `tx-sender`         |
 
+
+Contract implementers should take note that in Stacks 2.0, the memo field won't be included in the event emitted by successful `ft-transfer?` operations. As a consequence, if compliance with exchanges is a requirement, it is recommended to emit an event including the memo, by adding a `print` statement if the `ft-transfer?` is successful.
+
+Exemple:
```
Comment (hstove) [inline sips/sip-010/sip-010-fungible-token-standard.md:72]: Typo here

Comment (hstove) [inline sips/sip-010/sip-010-fungible-token-standard.md:72]:
```suggestion
Example:
```

**Review by hstove — APPROVED** [review]
Thanks Ludo!

**Comment by hstove** [comment]
@jcnelson this PR is a bit off because the SIP repo branch for `SIP-10` does not include the latest commits from my PR, and so this PR does not include those commits. I'm happy with merging this, but we should first sync my branch, then fix the one typo here, and then we can merge

File: sips/sip-010/sip-010-fungible-token-standard.md (hunk 1 above)
Comment (lgalabru) [reply sips/sip-010/sip-010-fungible-token-standard.md]: Updated, thanks!

**Comment by lgalabru** [comment]
Is this ball in my court, @hstove's, or @jcnelson's?

**Comment by jcnelson** [comment]
Either @hstove or @lgalabru -- it's the SIP authors' job to get the SIP into the right format.

**Comment by hstove** [comment]
Maybe a miscommunication - what I meant is that this PR is trying to merge into the `hstove-feat/sip-10-ft` branch, but the open PR (#5 ) has more recent commits than that branch. So this PR is not quite in sync with the latest of my PR.

Either way, it's all good - I've cherry-picked and pushed @lgalabru's commits to #5. So, I think we can close this PR!

**Pull request closed without merging**